pkg/reconciler/route/table_test.go

```diff
@@ -751,7 +751,6 @@ func TestReconcile(t *testing.T) {
 				},
 			),
 			simpleK8sService(Route("default", "becomes-local", WithConfigTarget("config"),
-				WithRouteLabel(map[string]string{network.VisibilityLabelKey: "cluster-local"}),
```

Comment:
I think the test is testing this combination:

current Route: not local, current Ingress: not local
new Route: local
=> new Ingress update: local

So I think the correct change is to change line 738 like what you did, but keep 754 the same